Code/Framework/AzCore/Platform/Common/VisualStudio/AzCore/Natvis/rapidjson.natvis

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
 	<!-- rapidjson::GenericValue - basic support -->
-	<Type Name="rapidjson_ly::GenericValue&lt;*,*&gt;">
+	<Type Name="rapidjson::GenericValue&lt;*,*&gt;">
```

Comment:
if we reverted to using the `rapidjson_ly` namespace, do we not want to use it here?

Reply:
nvm, I see we alias it using `namespace rapidjson = rapidjson_ly;`

Reply:
Hmm, this makes me think it might be better to use 
```
#define rapidjson rapidjson_ly
```
instead of 'using'.  With 'using' its a type alias and its unclear if the debugger / natvis can tell. 

I'll do a quick experiemnt and see!

Reply:
Confirmed as broken - the debugger shows it as the underlying type (rapidjson_ly) but the natvis system isn't smart enough to understand that rapidjson == rapidjson_ly..  I'll fix the natvis file